### PR TITLE
Packaging first version (1.0.0) of LibHyps.

### PR DESCRIPTION
**Pull request by Matafou**
LibHyps

This library defines a set of tactics to manipulate hypothesis
indivually or by group. In particular it allows to apply a tactic on
each hypothesis of a goal, or only *new* hypothesis after some tactic.
Manipulations include: automatic renaming, decomposition, extracting
premisses, but define your own manipulation on (groups of) hypothesis.

### CODE REVIEW

**Comment by Matafou** [comment]
Hi, These tests seem to fail quite early. Is this due to something wrong in my package?

File: coq-libhyps/coq-libhyps.1.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+license: "GPL3"
+
+build: [
+  ["./configure.sh"]
+  [make "-j%{jobs}%"]
+]
+
+install: [make "install"]
+
+remove: ["rm" "-R" "%{lib}%/coq/user-contrib/LibHyps"]
```
Comment (anton-trunov) [inline coq-libhyps/coq-libhyps.1.0.0/opam]: opam 2.0 tracks installed files, so you don't need `remove:` section

**Comment by Matafou** [comment]
Somehow I had files at the wrong place. This is fixed and also @anton-trunov remark.

**Comment by Matafou** [comment]
Thanks!

**Pull request merged**